datastore/tests/unit/test_entity.py

```diff
@@ -36,6 +36,7 @@ def test_ctor_defaults(self):
         entity = klass()
         self.assertIsNone(entity.key)
         self.assertIsNone(entity.kind)
+        self.assertIsNone(entity.id)
```

Comment:
@chemelnucfin IMO it's "bad form" to check a `@property` in the constructor test. I view these assertions as essentially checking whats in `entity.__dict__`

Reply:
I was just following what was there previously.

Reply:
Great. Does my approach seem logical?